src/steps/devices/converter.ts

```diff
@@ -33,6 +34,9 @@ export function createHostAgentEntity(device: Device): Entity {
         firewallEnabled: device['Firewall Enabled'] || undefined,
         fileVaultEnabled: device['FileVault Enabled'] || undefined,
         platform: device['OS Platform'] || undefined,
+        totalMemory: device['Total Memory (GB)'] || undefined,
+        totalDiskSpace: device['Total Disk Space (GB)'] || undefined,
+        lastOnline: parseTimePropertyValue(device['Last Online']) || undefined,
```

Comment:
Just FYI, `parseTimePropertyValue` already handles `undefined` values and will return `undefined` when `undefined` is supplied.